eclipse-collections/src/main/java/org/eclipse/collections/impl/list/mutable/CompositeFastList.java

```diff
@@ -178,7 +179,7 @@ public void reverseForEachWithIndex(ObjectIntProcedure<? super E> procedure)
     @Override
     public boolean isEmpty()
     {
-        return this.lists.allSatisfy(FastList<E>::isEmpty);
+        return this.lists.allSatisfy(AbstractRichIterable::isEmpty);
```

Comment:
This one is something else, but I like it and would turn it on in an inspection profile. Could you identify where it comes from?

Reply:
This came from: "Instance field access not qualified with 'this'"
Also, this was not an auto-fix, I had to convert it to anonymous inner class and then switch it to a method reference. It is a good inspection to have, I will raise a bug to Jetbrains, may be they can add it.